Makefile

```diff
@@ -114,8 +121,8 @@ test_go:
 		)
 
 test_e2e:
-	sed -i "s#image:.*$(IMAGE_NAME).*#image: $(DOCKER_IMAGE):$(BUILD_TAG)#" docker-compose.yaml
-	cd plugin/pki/e2e && ginkgo -v
+	sed -i -e "s#image:.*$(IMAGE_NAME).*#image: $(DOCKER_IMAGE):$(BUILD_TAG)#" docker-compose.yaml
+	cd plugin/pki/e2e && ~/go/bin/ginkgo -v
```

Comment:
we should, remove the path of the command, since in other OS or users may have  the path in different location.

Reply:
absolute path removed from ginkgo -v invocation